src/storage/txn/scheduler.rs

```diff
@@ -453,6 +460,23 @@ fn process_read(cid: u64, mut cmd: Command, ch: SyncSendCh<Msg>, snapshot: Box<S
     }
 }
 
+fn process_rawscan(snapshot: Box<Snapshot>,
+                   start_key: &Key,
+                   limit: usize,
+                   stats: &mut Statistics)
+                   -> Result<Vec<StorageResult<KvPair>>> {
```

Comment:
I want to reuse the ProcessResult here https://github.com/pingcap/tikv/blob/52693838cb0eec496d8c1268935bcea826844ce4/src/storage/txn/scheduler.rs#L69